templates/etc/snmp/snmpd.conf.j2

```diff
@@ -31,5 +31,11 @@ disk {{ snmpd_disk.path }} {{ snmpd_disk.threshold }}
 
 defaultMonitors         {{ 'yes' if snmpd_default_monitors else 'no' }}
 linkUpDownNotifications {{ 'yes' if snmpd_link_up_down_notifications else 'no' }}
+{% if snmpd_extensions is defined %}
+
+{% for snmpd_extension in snmpd_extensions %}
```

Comment:
`{% for snmpd_extension in snmpd_extensions | default([]) %}` so you can remove the `if`

Reply:
Done! :)